CMake/SlicerApplicationOptions.cmake

```
@@ -178,8 +178,8 @@ if(WIN32)
   # - Slicer_CPACK_NSIS_INSTALLER_WELCOME_FILE
   # - Slicer_CPACK_NSIS_INSTALLER_UNWELCOME_FILE
 
-  set(Slicer_CPACK_NSIS_INSTALLER_HEADER_FILE_DEFAULT "${Slicer_SOURCE_DIR}/Applications/Resources/Installer/Header.bmp")
-  set(Slicer_CPACK_NSIS_INSTALLER_WELCOME_FILE_DEFAULT "${Slicer_SOURCE_DIR}/Applications/Resources/Installer/WelcomeFinishPage.bmp")
+  set(Slicer_CPACK_NSIS_INSTALLER_HEADER_FILE_DEFAULT "${CMAKE_SOURCE_DIR}/Applications/${Slicer_MAIN_PROJECT}/Resources/Installer/Header.bmp")
+  set(Slicer_CPACK_NSIS_INSTALLER_WELCOME_FILE_DEFAULT "${CMAKE_SOURCE_DIR}/Applications/${Slicer_MAIN_PROJECT}/Resources/Installer/WelcomeFinishPage.bmp")
```

Good catch :100:

When I initially wrote this, I missed the fact you already created versions of `Header.bmp` and `WelcomeFinishPage.bmp` in https://github.com/KitwareMedical/SlicerCustomAppTemplate/pull/93

In this case, we could simply reuse `Slicer_APPLICATIONS_DIR` already set earlier.

```suggestion
  set(Slicer_CPACK_NSIS_INSTALLER_HEADER_FILE_DEFAULT "${Slicer_APPLICATIONS_DIR}/${Slicer_MAIN_PROJECT}/Resources/Installer/Header.bmp")
  set(Slicer_CPACK_NSIS_INSTALLER_WELCOME_FILE_DEFAULT "${Slicer_APPLICATIONS_DIR}/${Slicer_MAIN_PROJECT}/Resources/Installer/WelcomeFinishPage.bmp")
```